### PR TITLE
Please refer to ticket : PEN-1167

### PR DESCRIPTION
**Pull request by mahditasfin**

### CODE REVIEW

File: frontend/src/components/penreg/student-search/StudentSearchDisplay.vue [hunk 1]
```diff
@@ -346,6 +327,7 @@ export default {
     prepPut(studentSearchFilters) {
       let sort = {};
       sort[this.headerSortParams.currentSort] = this.headerSortParams.currentSortAsc ? 'ASC' : 'DESC';
+      console.log(this.pageNumber);
```
Comment (mishraomp) [inline frontend/src/components/penreg/student-search/StudentSearchDisplay.vue:330]: could you please remove the console log

File: frontend/src/components/penreg/student-search/StudentSearchDisplay.vue [hunk 2]
```diff
@@ -310,7 +291,7 @@ export default {
       if(validationRequired === false || (this.$refs.studentSearchForm.validate() && this.searchHasValues())) {
         const studentSearchKeys = Object.keys(this.studentSearchParams).filter(k => (this.studentSearchParams[k] && this.studentSearchParams[k].length !== 0));
         let studentSearchFilters;
-        if (studentSearchKeys && studentSearchKeys.length > 0) {
+        if (studentSearchKeys && studentSearchKeys.length >1) {
```
Comment (mishraomp) [inline frontend/src/components/penreg/student-search/StudentSearchDisplay.vue:294]: we could do if(studentSearchKeys?.length >1)

Comment (mahditasfin) [reply frontend/src/components/penreg/student-search/StudentSearchDisplay.vue:294]: I will fix those now, thank you Om

File: frontend/src/components/penreg/student-search/StudentSearchDisplay.vue [hunk 3]
```diff
@@ -345,7 +326,7 @@ export default {
     },
     prepPut(studentSearchFilters) {
       let sort = {};
-      sort[this.headerSortParams.currentSort] = this.headerSortParams.currentSortAsc ? 'ASC' : 'DESC';
+      sort[this.headerSortParams.currentSort] = this.headerSortParams.currentSortAsc ? 'ASC' : 'DESC'
```
Comment (mightycox) [inline frontend/src/components/penreg/student-search/StudentSearchDisplay.vue:329]: Removed semicolon

**Pull request merged**